src/core.js

```diff
@@ -56,11 +58,14 @@ extend( QUnit, {
 			var module = {
 				name: moduleName,
 				parentModule: parentModule,
-				tests: []
+				tests: [],
+				testsRun: 0
```

Comment:
this might be my only concern on this PR. Exposing a new detail on the API conflicts with the plans for the EventEmitter changes (#882).

Although, the `tests` array [will still prevail](https://github.com/js-reporters/js-reporters/issues/30) and I wonder if we can set an equivalent property on the [reference object for the test on that array](https://github.com/jquery/qunit/pull/919/files#diff-f6c2ded91c9881cdc31b5bfcabb54c3bR25). Doing that, we would have the tradeoff to filter the array, but it would save us a new API exposure.